pkg/analysis/utils/serialization/testdata/src/pointers_always/arrays.go.golden

```diff
@@ -0,0 +1,43 @@
+package a
+
+type TestArrays struct {
+	Array []string `json:"array,omitempty"` // want "field Array should have the omitempty tag."
+
+	ArrayWithOmitEmpty []string `json:"arrayWithOmitEmpty,omitempty"`
+
+	ArrayPtr []*string `json:"arrayPtr,omitempty"` // want "field ArrayPtr should have the omitempty tag."
+
+	ArrayWithOmitEmptyPtr []*string `json:"arrayWithOmitEmptyPtr,omitempty"`
+
+	// +kubebuilder:validation:MinItems=1
+	ArrayWithPositiveMinItems []string `json:"arrayWithPositiveMinItems,omitempty"` // want "field ArrayWithPositiveMinItems should have the omitempty tag."
+
+	// +kubebuilder:validation:MinItems=1
+	ArrayWithPositiveMinItemsWithOmitEmpty []string `json:"arrayWithPositiveMinItemsWithOmitEmpty,omitempty"`
+
+	// +kubebuilder:validation:MinItems=0
+	ArrayWithZeroMinItems []string `json:"arrayWithZeroMinItems,omitempty"` // want "field ArrayWithZeroMinItems should have the omitempty tag."
+
+	// +kubebuilder:validation:MinItems=0
+	ArrayWithZeroMinItemsWithOmitEmpty []string `json:"arrayWithZeroMinItemsWithOmitEmpty,omitempty"`
+
+	ByteArray []byte `json:"byteArray,omitempty"` // want "field ByteArray should have the omitempty tag."
+
+	ByteArrayWithOmitEmpty []byte `json:"byteArrayWithOmitEmpty,omitempty"`
+
+		// +kubebuilder:validation:MinLength=1
+	ByteArrayWithMinLength []byte `json:"byteArrayWithMinLength,omitempty"` // want "field ByteArrayWithMinLength should have the omitempty tag."
+
+	// +kubebuilder:validation:MinLength=1
+	ByteArrayWithMinLengthWithOmitEmpty []byte `json:"byteArrayWithMinLengthWithOmitEmpty,omitempty"`
+
+	// +kubebuilder:validation:MinLength=0
+	ByteArrayWithMinLength0 []byte `json:"byteArrayWithMinLength0,omitempty"` // want "field ByteArrayWithMinLength0 should have the omitempty tag."
+
+	// +kubebuilder:validation:MinLength=0
+	ByteArrayWithMinLength0WithOmitEmpty []byte `json:"byteArrayWithMinLength0WithOmitEmpty,omitempty"`
+
+    PtrArray []string `json:"ptrArray,omitempty"` // want "field PtrArray should have the omitempty tag." "field PtrArray underlying type does not need to be a pointer. The pointer should be removed."
```

Comment:
Indentation seems off in a couple places here - is this an issue with the suggestion logic?

Reply:
Possibly, will double check the indentation is consistent, I think my IDE would normally fix it up but it doesn't recognise that this is go, because of the golden extension